moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -77,6 +78,11 @@ TrajectoryExecutionManager::TrajectoryExecutionManager(const robot_model::RobotM
   if (!node_handle_.getParam("moveit_manage_controllers", manage_controllers_))
     manage_controllers_ = false;
 
+  ROS_WARN_NAMED("trajectory_execution_manager",
+                 "Deprecation warning: the namespace for params ~allowed_execution_duration_scaling and "
+                 "~allowed_goal_duration_margin are changed under ns `~/trajectory_execution`."
+                 "Parameter reading from that under old ns will be deprecated.");
+
   if (!node_handle_.getParam("allowed_execution_duration_scaling", allowed_execution_duration_scaling_))
```

Comment:
The warning should only be issues when the parameter was found in the _wrong_ namespace.